content/general/use-latest-version-of-everything.md

```diff
@@ -0,0 +1,31 @@
+---
+title: use latest version of everything
+author: 
+  name: Billy Lando
+  url: https://github.com/billyjov
+---
+
+# Problem 
+
+The Angular team and community are continually improving the ecosystem to make it easier to build applications. Both the performance and the compiler (e.g Ivy Renderer) are constantly being improved for better web applications. 
+
+Angular uses semantic versioning (semver) which means they use a regular schedule of releases. This includes a major release every six month, 1-3 minor releases for each major release, and a patch release almost every week. It's important to keep up with major releases as they contain significant new features. The longer we wait to update our application, the more expensive will be a future update. Be aware, that major releases may contain breaking changes.   
+
+In addition, when API's get deprecated they remain present in the next two major releases until they are removed. Again, if we wait to long, it's likely that the update requires much more work. Deprecations hints are available in the [Changelog](https://github.com/angular/angular/blob/master/CHANGELOG.md).
+
+# Solution
+
+You can follow this steps using Angular CLI:
+
+- **Step 1:** Create a new feature branch  
+- **Step 2:** Run `ng update @angular/core @angular/cli` inside your project directory
+- **Step 3:** Run `ng serve`, `ng test`, `ng build --prod` and make sure your app works as expected 
+- **Step 4:** Fix update deprecations, issues, styling issues in case of Angular Material and run the previous step again
+- **Step 5:** merge or rebase your changes on top of the main branch
+
+For more informations, check out the [official update guide](https://update.angular.io/) on how to update from different versions.
```

Comment:
For more `information` instead of `informations`.

Reply:
Thanks..I will learn to write correctly over time :)